install_workflow/deploy_workflow.sh

```diff
@@ -30,7 +30,7 @@ touch ${ROOT}/share/bashrc.uceq
 
 echo "export gmsim='$ROOT'" >> ${ROOT}/share/bashrc.uceq
 echo "export nobackup='/nesi/nobackup/nesi00213'" >> ${ROOT}/share/bashrc.uceq
-echo 'export PATH=$PATH:'${ROOT}/workflow/scripts >> ${ROOT}/share/bashrc.uceq
+echo 'export PATH=$PATH:'${ROOT}/workflow/scripts:/nesi/project/nesi00213/opt/mahuika/ffmpeg_build/bin:/nesi/project/nesi00213/opt/maui/gmt/5.4.4/bin >> ${ROOT}/share/bashrc.uceq
```

Comment:
Have to be careful about machine dependance. I'm pretty sure we only use the mahuika GMT binary.